cmd/gateclient/client.go

```diff
@@ -31,19 +31,20 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/spinnaker/spin/config"
+	config2 "github.com/spinnaker/spin/config/auth/iap"
```

Comment:
Is there a particular reason this is aliased? Shouldn't this resolve to the `iap` package? If the alias is necessary I'd name it `iap` or something similar.

Reply:
changed the package name to `iap`